.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
```

Comment:
Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

Reply:
i changed the file that you specified me 👍 (ln 11, 23, 37)
![changes in yml file](https://user-images.githubusercontent.com/97492524/218516478-04da9843-915a-4303-8e24-88d8f3fd22de.png)

the linters are runnings well
![hint](https://user-images.githubusercontent.com/97492524/218516849-9279ebff-183d-4803-9331-efd02e398791.png)
![stylelint](https://user-images.githubusercontent.com/97492524/218516879-2628bc4a-0fb4-4f3a-bb0b-7bac294b607c.png)